src/components/__tests__/HelpButton.cy.js

```diff
@@ -0,0 +1,217 @@
+import HelpButton from 'components/HelpButton.vue';
+import { i18n } from '../../boot/i18n';
+
+const rideToWorkByBikeConfig = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG
+);
+const colorPrimary = rideToWorkByBikeConfig.colorPrimary;
+
+describe('<HelpButton>', () => {
+  it('has translation for all strings', () => {
+    cy.testLanguageStringsInContext([], 'index.help', i18n);
+  });
+
+  context('button grey', () => {
+    beforeEach(() => {
+      cy.mount(HelpButton, {
+        props: {
+          color: 'grey-10',
+          size: '13px',
+        },
+      });
+      cy.viewport('macbook-16');
+    });
+
+    it('renders help button', () => {
+      cy.window().then(() => {
+        cy.dataCy('button-help').should('be.visible');
```

Comment:
Same as my comment above https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/51/files#r1364873399.

Reply:
Edited.